pkg/cmd/util/labels.go

```diff
@@ -0,0 +1,35 @@
+package util
+
+import (
+	"fmt"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/labels"
+)
+
+// ParseLabel parses a label spec into a Set
+func ParseLabel(labelSpec string) (labels.Set, error) {
+	args := strings.Split(labelSpec, "=")
+
+	switch len(args) {
+	case 2:
+		return map[string]string{args[0]: args[1]}, nil
+	default:
+	}
+
+	return nil, fmt.Errorf("invalid label spec: %s", labelSpec)
+}
+
+var (
+	mut     sync.Mutex
+	version int
+)
+
+func GetLabel() string {
```

Comment:
To clarify, create should only ever create what you send it.  No changes.

----- Original Message -----
> No..... don't do that.
> 
> ----- Original Message -----
> > > +
> > > +	switch len(args) {
> > > +	case 2:
> > > +		return map[string]string{args[0]: args[1]}, nil
> > > +	default:
> > > +	}
> > > +
> > > +	return nil, fmt.Errorf("invalid label spec: %s", labelSpec)
> > > +}
> > > +
> > > +var (
> > > +	mut     sync.Mutex
> > > +	version int
> > > +)
> > > +
> > > +func GetLabel() string {
> > 
> > it's not just new-app right?  "osc create -f myconfiglist.json" also needs
> > labels applied.
> > 
> > 
> > ---
> > Reply to this email directly or view it on GitHub:
> > https://github.com/openshift/origin/pull/1537/files#r27611732
>